client.go

```diff
@@ -0,0 +1,244 @@
+}
+
+// Exists returns the number of key arguments that exist.
+// Note that if the same existing key is mentioned in the argument
+// multiple times, it will be counted multiple times.
+func (c *Client) Exists(keys ...string) (int64, error) {
+	return c.redisClient.Exists(c.vu.Context(), keys...).Result()
+}
+
+// Incr increments the number stored at `key` by one. If the key does
+// not exist, it is set to zero before performing the operation. An
+// error is returned if the key contains a value of the wrong type, or
+// contains a string that cannot be represented as an integer.
+func (c *Client) Incr(key string) (int64, error) {
+	return c.redisClient.Incr(c.vu.Context(), key).Result()
+}
+
+// IncrBy increments the number stored at `key` by `increment`. If the key does
+// not exist, it is set to zero before performing the operation. An
+// error is returned if the key contains a value of the wrong type, or
+// contains a string that cannot be represented as an integer.
+func (c *Client) IncrBy(key string, increment int64) (int64, error) {
+	return c.redisClient.IncrBy(c.vu.Context(), key, increment).Result()
+}
+
+// Decr decrements the number stored at `key` by one. If the key does
+// not exist, it is set to zero before performing the operation. An
+// error is returned if the key contains a value of the wrong type, or
+// contains a string that cannot be represented as an integer.
+func (c *Client) Decr(key string) (int64, error) {
+	return c.redisClient.Decr(c.vu.Context(), key).Result()
+}
+
+// DecrBy decrements the number stored at `key` by `decrement`. If the key does
+// not exist, it is set to zero before performing the operation. An
+// error is returned if the key contains a value of the wrong type, or
+// contains a string that cannot be represented as an integer.
+func (c *Client) DecrBy(key string, decrement int64) (int64, error) {
+	return c.redisClient.DecrBy(c.vu.Context(), key, decrement).Result()
+}
+
+// RandomKey returns a random key.
+func (c *Client) RandomKey() (string, error) {
+	return c.redisClient.RandomKey(c.vu.Context()).Result()
+}
+
+// Mget returns the values associated with the specified keys.
+func (c *Client) Mget(keys ...string) ([]interface{}, error) {
+	return c.redisClient.MGet(c.vu.Context(), keys...).Result()
+}
+
+// Expire sets a timeout on key, after which the key will automatically
+// be deleted.
+// Note that calling Expire with a non-positive timeout will result in
+// the key being deleted rather than expired.
+func (c *Client) Expire(key string, seconds int) (bool, error) {
+	return c.redisClient.Expire(c.vu.Context(), key, time.Duration(seconds)*time.Second).Result()
+}
+
+// Ttl returns the remaining time to live of a key that has a timeout.
+// nolint:stylecheck,revive
+func (c *Client) Ttl(key string) (time.Duration, error) {
+	d, err := c.redisClient.TTL(c.vu.Context(), key).Result()
+	if err != nil {
+		return 0, err
+	}
+
+	return time.Duration(d.Seconds()), nil
```

Comment:
```suggestion
func (c *Client) Ttl(key string) (float64, error) {
	d, err := c.redisClient.TTL(c.vu.Context(), key).Result()
	if err != nil {
		return 0, err
	}

	return d.Seconds(), nil
```
What do you think about removing duration since TTL is always seconds?

Reply:
Fair point, but I'll return an `int64` instead: the underlying type of `Duration`, also because we use seconds we don't need the floating point precision 🤝